fizzbuzz.js

```diff
@@ -0,0 +1,80 @@
+//Really, really inefficient way to do FizzBuxx
+
+//Variable declaration
+var maxNum = 100;
+var primeArray = [];
+var fizzBuzzArray = [];
+var tempArray = [];
+var i = 2;
+var x;
+var prime;
+var prime2;
+var count = 0;
+var print;
+
+//Calculates all prime numbers between 2 and maxNum (inclusive on both ends)
+while (i <= maxNum) {
+    prime = true;
+    for (x in primeArray) {
```

Comment:
You should never use `for...in` to iterate over arrays in JavaScript.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in#Array_iteration_and_for...in

You are welcome to use [`for...of`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of) if you don't need to [support IE 11 and below](http://kangax.github.io/compat-table/es6/#test-for..of_loops), but a regular `for` loop or `primeArray.forEach(...)` is preferred.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach